### PR TITLE
[fix bug 1115125] Missing favicon in tab display for some of the categories

### PR DESCRIPTION
**Pull request by yashmurty**
Fixes the issue of missing favicon.

### CODE REVIEW

File: remo/base/static/base/js/remolib.js [hunk 1]
```diff
@@ -36,6 +36,14 @@ function hash_set_value(key, value) {
     }
 
     $(location).attr('hash', hash);
+
+    (function() {
+        var link = document.createElement('link');
+        link.type = 'image/x-icon';
+        link.rel = 'shortcut icon';
+        link.href = 'https://reps.mozilla.org/static/base/img/remo/shortcut.ico';
```
Comment (johngian) [inline remo/base/static/base/js/remolib.js]: Please replace this hard coded URL with ``/static/base/img/remo/shortcut.ico``.

File: remo/base/static/base/js/remolib.js [hunk 2]
```diff
@@ -36,6 +36,14 @@ function hash_set_value(key, value) {
     }
 
     $(location).attr('hash', hash);
+
+    (function() {
+        var link = document.createElement('link');
```
Comment (johngian) [inline remo/base/static/base/js/remolib.js]: Creating a new link on each ``hash`` change leads to multiple duplicate links in ``head``. I attached a screenshot after some hash changes.
selection_021

Comment (johngian) [inline remo/base/static/base/js/remolib.js]: Instead you could find the link and replace it:
```
(function() {
    var url = $("link[rel=shortcut icon]").remove().attr("href");
    var favicon = document.createElement('link');
    favicon.type = 'image/x-icon';
    favicon.rel = 'shortcut icon';
    favicon.href = url;
    favicon.appendTo('head');
}());

**Comment by johngian** [comment]
Hey @yashmurty 
Thank you for the contribution. I've added a couple of comments and after you fix that your PR is ready to be merged.

Thank you :rocket:

**Comment by yashmurty** [comment]
Hey @johngian,

I have made the recommended changes. 
Please let me know if there is anything else that I should fix.

Thanks :)

**Comment by johngian** [comment]
Closing this one after #1039

**Pull request closed without merging**